api/loader.go

```diff
@@ -69,13 +68,19 @@ func (m *Loader) RegisterApi(referenceSpec *Spec) {
 	}
 
 	if skip {
-		hasher := speedbump.PerSecondHasher{}
-		limit := referenceSpec.RateLimit.Limit
-		limiter := speedbump.NewLimiter(m.redisClient, hasher, limit)
-
 		var handlers []router.Constructor
 		if referenceSpec.RateLimit.Enabled {
-			handlers = append(handlers, limitter.NewRateLimitMiddleware(limiter, hasher, limit).Handler)
+			rate, err := limiter.NewRateFromFormatted(referenceSpec.RateLimit.Limit)
```

Comment:
I am not totally sure if this is a breaking change. At least my registered routes for api-v2 (locally) break and my whole app crashes.